docs/versioned_docs/version-v3/configuration/options.md

```diff
@@ -462,6 +462,15 @@ cookies: {
       path: '/',
       secure: useSecureCookies
     }
+  },
+  nonce: {
```

Comment:
This is v3 docs, you'll need to update v4 instead 💡 
docs/docs/configuration/options.md

Reply:
good spot, my mistake will update that now.

Reply:
Fixed